src/modules/ordersTable/pure/OrdersTableContainer/OrdersTable.tsx

```diff
@@ -308,6 +309,20 @@ export function OrdersTable({
     checkbox.checked = allOrdersSelected
   }, [allOrdersSelected, selectedOrders.length])
 
+  const [collapsedOrder, setCollapsedOrder] = useState<{ [index: string]: boolean }>({})
+
+  const handleOnClickToggle = (order: ParsedOrder) => {
+    const orderId = order.composableCowInfo?.id // If it's a parent, use its own ID
+    const parentOrderId = getComposableCowParentId(order) // If it's a child, use its parent ID
+
+    const idToToggle = parentOrderId !== undefined ? parentOrderId : orderId
+
+    // If the order has a parent, toggle the collapsed state for that parent
```

Comment:
IMO, this comment is not needed, since you already stated this in the `idToToggle` declaration